modules/dnn/src/net.cpp

```diff
@@ -115,12 +115,12 @@ void Net::forward(std::vector<std::vector<Mat>>& outputBlobs,
 }
 
 // FIXIT drop from inference API
-Net Net::quantize(InputArrayOfArrays calibData, int inputsDtype, int outputsDtype)
+Net Net::quantize(InputArrayOfArrays calibData, int inputsDtype, int outputsDtype, bool perTensor = false)
```

Comment:
> bool perTensor = false

Default value works properly from .hpp files only. They are useless in .cpp files

Reply:
Thanks, fixed.